refs/schemas/priv/expected-behavior.md

```diff
@@ -0,0 +1,655 @@
+- Regardless of restrictions
+    - `OTHER` Demands: recommend human review and status = `UNDER-REVIEW`
+    - `TRANSPARENCY.KNOWN` Demands: recommend status = `GRANTED`, and data = `YES`.
+    - `TRANSPARENCY.DATA-CATEGORIES` Demands: recommend status = `GRANTED`, and data = list of Data Categories that are included in any of the [Privacy Scope Triples](#privacy-scope-triples) included in the Eligible Privacy Scope.
+    - `TRANSPARENCY.PROVENANCE`, see [Resolving provenance in requests](#resolving-provenance-in-requests)
+    - `TRANSPARENCY.ORGANIZATION`, `TRANSPARENCY.POLICY`, `TRANSPARENCY.RETENTION`, `TRANSPARENCY.DPO`, `TRANSPARENCY.WHERE`, `TRANSPARENCY.WHO`, Demands: recommend status = `GRANTED`, and data = information corresponding to the request taken from configuration settings as defined under [Implications for Systems](./RFC-PRIV.md#design-implications-for-systems-implementing-PRIV) and under [Configuration and Prerequisites](#configuration-and-prerequisites).
+
+- With regards to Demand restrictions (if any) limiting the scope of the Demand:
+    - first, check for presence of incompatible restrictions (and if incompatible recommend status = `DENIED`, motive = `REQUEST-UNSUPPORTED`):
+        - [Consent Restriction](./RFC-PRIV.md#consent-restriction) with any other type of Restriction,
+        - [Consent Restriction](./RFC-PRIV.md#consent-restriction) within a Demand other than `REVOKE-CONSENT`,
+        - More than one [Capture Restriction](./RFC-PRIV.md#capture-restriction),
+        - More than one [Data Reference Restriction](./RFC-PRIV.md#data-reference-restriction)
+        - A Capture Restriction and any other restriction that is not a Privacy Scope Restriction, or a Data Reference Restriction,
+        - More than one [Date Range](./RFC-PRIV.md#date-range) Restriction.
+
+    - second, Calculate **Restriction Scope** and **Concerned Fragments**. This is done by processing every Restriction according to the following approach:
+        - at the beginning set **Restriction Scope** to be equal to the Eligible Privacy Scope of the Data Subject
+        **Concerned Fragments** is set to all Data Capture Fragments the `scope`s of which are included in the **Restriction Scope**.
+
+        - when processing a [Privacy Scope](./RFC-PRIV.md#privacy-scope) Restriction, set the new **Restriction Scope** to be the intersection of the previous **Restriction Scope** with the Privacy Scope of the Restriction.
+        **Concerned Fragments** is set to all Data Capture Fragments the `scope`s of which are included in the **Restriction Scope**.
+
+        - when processing a [Capture Restriction](./RFC-PRIV.md#capture-restriction) set the new **Restriction Scope** to be the intersection of the previous **Restriction Scope** with union of the [Privacy Scope](./RFC-PRIV.md#privacy-scope)s of each Data Capture Fragment of each Data Capture the `capture-id` of which is listed under `capture-ids` of that Capture Restriction
```

Comment:
Doing union is redundant. If a triple `f` (one triple of a fragment) belongs to `F` (union) as well as to intersection `F ∩ R`, that means `f` it belongs to `R` (restriction scope) as well.

Reply:
wait,

Restriction Scope (RS) = 
    *intersection* (
    Previous Restriction Scope (PRS), 
    *union* (
        `privacy scope` of each Data Capture Fragment associated with Data Capture where `capture-id`=one of the `capture-ids` of the restriction
        )
    )

Where is the problem? You don't have to do the union operation. You can implement is as a loop over all fragments of all captures of the restriction. It is equivalent, I think.

Reply:
1.
The new RS will be a subset of the PRS. On the next potential restriction step, a new RS will be applied and some Fragments will be missed (compared to if the next step was applied before the Capture Restriction, with PRS).
That makes Restriction not commutative.

2.
Doing union + intersection is redundant and compute-intensive.
Same fragments will be selected if filtered by `PRS + capture-id` or `RS`.

Reply:
Restriction-scope is not about the fragments at all (that is why we MUST separately comptue the Concerned Fragments)
By definition, [union is commutative and distributive](https://en.wikipedia.org/wiki/Union_(set_theory)#Algebraic_properties)

Let's have a call to clarify this. We are getting nowhere here.

Reply:
I see where the problem is. What you want to say is that you can calculate the Concerned Fragmnets directly (PRS + capture-id) without ever updating Restriction Scope.

This would have been better, compation-wise, but the issue is that you also need to have the Restriction Scope. 
The Data Subject can make a Capture Restriction on a `TRANSPARENCY.PROCESSING-CATEGORIES`. You must be able to deliver this informaiton (from the restriction scope). 

And you want all the fragments of a capture affect the Restriction Scope (hence the Union).